### ISSUE TITLE
Question : How to do action on detection

### ISSUE THREAD

**Issue by benda95280** [ethaniccc/Mockingbird#145]
Dear,

How it could be possible to do actions on detection ? (Like write in database some informations ...) 
It seems there is no events ?

Regards,

**Comment by mmm545** [comment]
yes there isn't (there are logs but those aren't saved on disk iirc)
you would probably want to edit the code yourself to save information in a database when a player fails a check

**Comment by benda95280** [comment]
@mmm545 Thanks for the answer, should i work only here ?
https://github.com/ethaniccc/Mockingbird/blob/acaf096b34f49a1792354e926324a19e8ab65b1c/src/ethaniccc/Mockingbird/detections/Detection.php#L119
